### DISCUSSION TITLE
how to select a P2P server forcefully.

### DISCUSSION THREAD

**Discussion by grrminator** [bubuntux/nordlynx#98] [Q&A]
I would prefer to select P2P servers through nordlynx, but am unsure how to. Can anyone assist. I am a bit new to docker. I managed to get it to connect to regular servers just letting it autoselect, but would like it to use P2P if at all possible.

**Comment by tmknight** [comment]
You'll likely find it unnecessary, however:
`QUERY: filters\[servers_groups\]\[identifier\]=legacy_p2p`

Here's a link that might provide more insight: [how-to-use-public-nordvpn-api](https://sleeplessbeastie.eu/2019/02/18/how-to-use-public-nordvpn-api/)
